spec/lib/google_maps_geocoder_spec.rb

```diff
@@ -11,7 +11,9 @@
         GoogleMapsGeocoder.new('White House')
       rescue SocketError
         pending 'waiting for a network connection'
-      rescue GoogleMapsGeocoder::GeocodingError
+      rescue GoogleMapsGeocoder::GeocodingError => e
+        raise e if e.json['status'] == 'REQUEST_DENIED'
```

Comment:
```suggestion
        raise if e.json['status'] == 'REQUEST_DENIED'
```

The `raise` without an argument can be used when the error to raise is the same as the one caught.